cmd/pipeline-template/pipeline_template.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2018, Google, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package pipeline_template
+
+import (
+	"github.com/spf13/cobra"
+	"io"
+)
+
+type pipelineTemplateOptions struct{}
+
+var (
+	pipelineTemplateShort   = ""
+	pipelineTemplateLong    = ""
+	pipelineTemplateExample = ""
+)
+
+func NewPipelineCmd(out io.Writer) *cobra.Command {
```

Comment:
Why not `newPipelineTemplateCmd`?

Reply:
Good catch, I made a few more changes like that as well.